### PR TITLE
Network service-tests use framework helper

### PR DESCRIPTION
**Pull request by breakreturn**

### CODE REVIEW

File: servicetest/networkgateway/test_network_gateway.py [hunk 1]
```diff
@@ -226,8 +250,5 @@ def test_ip_diversity(self):
             assert 0 == testhelper.compare_ips(CURRENT_DIR, "f1", "f2")
```
Comment (kursatkobya) [inline servicetest/networkgateway/test_network_gateway.py:252]: This can create problems later.

**Review by tobsan — CHANGES_REQUESTED** [review]
Can we please remove google.com from these tests. If we want to use something else than example.com, we can use example.org or example.net, see https://tools.ietf.org/html/rfc2606#section-3

File: servicetest/testframework/testhelper.py [hunk 2]
```diff
@@ -150,14 +198,48 @@ def __file_name(self):
...
-        h = EnvironmentHelper(file_base_path)
+        env_vars_file_base_path = env_vars_file_base_path_parsed_value.pop()
+        h = EnvironmentHelper(env_vars_file_base_path)
         # Get the environment from inside the container
         env_vars = h.get_env_vars()
         # Dump the information for the helper to read back later in the tests
         h.write_result(env_vars)
+
+    do_ping_file_base_path_parsed_value = getattr(args, DO_PING_OPTION)
+    if do_ping_file_base_path_parsed_value is not None:
```
Comment (DunderRoffe) [inline servicetest/testframework/testhelper.py]: really hard to read as the variables due to the combination: long and similar. Consider dropping the prefix and add a comment instead.

**Review by DunderRoffe — CHANGES_REQUESTED** [review]

File: servicetest/testframework/testhelper.py (hunk 2 above)
Comment (breakreturn) [reply servicetest/testframework/testhelper.py]: I agree. I think this structure will need to change anyway, when and if we can make use of this generic helper design. But point taken, I will try to find some better names or use whitespace for an easier time for the eye.

File: servicetest/networkgateway/test_network_gateway.py (hunk 1 above)
Comment (breakreturn) [reply servicetest/networkgateway/test_network_gateway.py:252]: Added a comment that this is broken at the moment.

**Review by DunderRoffe — APPROVED** [review]

**Review by tobsan — APPROVED** [review]

**Pull request merged**